.github/workflows/deploy_config.yaml

```diff
@@ -87,8 +87,6 @@ jobs:
         ref: "main"
         path: "production-pipelines"
         submodules: recursive
-        # Comment this line out to revert to using the default $GITHUB_TOKEN.
-        token: ${{ secrets.PRODUCTION_PIPELINES_RAW_REPO_TOKEN }}
```

Comment:
I think this is the one I was expecting to stop working when this token expired. The others can fall back to unauthenticated access to now-public repositories, but not this one that's explicitly setting an action's `token` parameter.

So that this remains a hint for next time we have to make that repo private (hopefully never happens, but it's hard to make predictions about the future), I'd rather this was just commented out and the comment I guess updated to something like _Uncomment this line and set up a PAT if production-pipelines needs to be taken private_.

Reply:
fair comment, I'll do that instead!